dnf-docker-test/features/steps/test_behave.py

```diff
@@ -93,6 +93,26 @@ def given_repo_condition(context, repo):
         f.write('[' + repo + ']\nname=' + repo + '\nbaseurl=http://127.0.0.1/repo/' + repo + '\nenabled=1\ngpgcheck=0')
 
 
+@step('I execute rpm command "{command}" from repository "{repository}"')
```

Comment:
Is there some possibility for forced installing rpm with broken dependencies via dnf?